medical-portal/src/UI/src/app/dashboard/dashboard.component.ts

```diff
@@ -195,6 +196,7 @@ export class DashboardComponent {
       data: searchedCase,
     });
     dialogRef.afterClosed().subscribe((result) => {
+      this.getClaimedDmerCases();
```

Comment:
for future consideration, we could optimize this by avoiding a second database query. But for now, this a good quick solution